README.md

```diff
@@ -1 +1,46 @@
-# portfolio-mobile
\ No newline at end of file
+# portfolio-mobile
+ This project is the third milestone of the portfolio mobile version
+
+
+## Built With
+
+- Major languages: HTML & CSS
+- A Linter for HTML & CSS
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+### Install : 
+A Linter for HTML & CSS
+
+
+
+
+## Author
+
+- GitHub: https://github.com/Piercel2022
+- Twitter: https://twitter.com/pier_celestin
+- LinkedIn: https://linkedin.com/in/piercelestinmax
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] I think it would be better to fix this broken link to make your documentation more professional. I suggest you add the `MIT.md` file in your root directory and this will fix the issue. You can get it from [here](https://github.com/microverseinc/readme-template/blob/master/MIT.md).